src/main/java/nova/core/render/pipeline/ConnectedTextureRenderPipeline.java

```diff
@@ -51,7 +53,7 @@
 	 * Each bit corresponds to a direction.
 	 * E.g: 000011 will render top and bottom
 	 */
-	public int faceMask = 0xff;
+	public Predicate<Direction> faceMask = dir -> true;
```

Comment:
It's not a mask though now is it. Can you think up of a better name? I cam up with `faceFilter` or `facePredicate`.